dev/app/footer/footer/container.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import React,{Component} from 'react';
+import {connect} from 'react-redux';
+
+import FooterComp from './component';
+// maybe import actions to click
+
+class Footer extends Component{
+    render(){
+        return(
+            <FooterComp />
+        )
+    }
+}
+
```

Comment:
Add mapStateToProps and mapDispatchToProps for consistency even if you don't have anything to pass.